signer/src/error.rs

```diff
@@ -74,6 +74,10 @@ pub enum Error {
     #[error("Failed to parse the stacks.api portion of the config: {0}")]
     StacksApiConfig(#[source] config::ConfigError),
 
+    /// Error when reading the stacks API part of the config.toml
+    #[error("Could not transform sepc256k1::PublicKey to stacks variant: {0}")]
+    StacksPublicKey(&'static str),
```

Comment:
I see this stems from the `from_slice` function using static string slices as the error type. Ugh.

Reply:
Oh and I need to update the comment for the variant.